pennylane/gradients/general_shift_rules.py

```diff
@@ -20,20 +22,26 @@
 import pennylane as qml
 
 
-def process_shifts(rule, tol=1e-10, check_duplicates=True):
+def process_shifts(rule, tol=1e-10, batch_duplicates=True):
     """Utility function to process gradient rules.
 
     Args:
-        rule (array): a ``(N, M)`` array corresponding to ``M`` terms
+        rule (array): a ``(M, N)`` array corresponding to ``M`` terms
             with parameter shifts. ``N`` has to be either ``2`` or ``3``.
-            The first row corresponds to the linear combination coefficients;
-            the last row contains the shift values.
-            If ``N=3``, the middle row contains the multipliers.
+            The first column corresponds to the linear combination coefficients;
+            the last column contains the shift values.
+            If ``N=3``, the middle column contains the multipliers.
         tol (float): floating point tolerance used when comparing shifts/coefficients
             Terms with coefficients below ``tol`` will be removed.
-        check_duplicates (bool): whether to check the input ``rule`` for duplicate
+        batch_duplicates (bool): whether to check the input ``rule`` for duplicate
             shift values in its second row.
 
+    Returns:
+        array: The processed shift rule with small entries rounded to 0, sorted
+        with respect to the absolute value of the shifts, and groups of shift
+        terms with identical (multiplier and) shift fused into one term each,
+        if ``batch_duplicates=True``.

```

Comment:
There was no returns? 😱

Reply:
Oh wait, it is a private function 😂

Reply:
Haha yes, more of a pet peeve of mine :D